backend/de.metas.swat/de.metas.swat.base/src/main/java/org/adempiere/inout/util/ShipmentScheduleQtyOnHandStorage.java

```diff
@@ -214,14 +216,17 @@ private boolean hasStockDetails()
 		return !stockDetails.isEmpty();
 	}
 
-	public ShipmentScheduleAvailableStock getStockDetailsMatching(@NonNull final I_M_ShipmentSchedule sched)
+
+	@Override
+	public List<ShipmentScheduleAvailableStockDetail> getStockDetailsMatching(final @NonNull OlAndSched olAndSched)
```

Comment:
why do i need OlAndSched here?

Reply:
nvm

Reply:
Not in this implementation of the method, but I assumed the other implementation should need to reference at least the orderlineLine. But that may no longer be the case, since the lookup by reservation is already done. Reverted for now, will change it back again if found needed during actual implementation.